Loop/Managers/NotificationManager.swift

```diff
@@ -67,14 +67,6 @@ extension NotificationManager {
         center.requestAuthorization(options: authOptions) { (granted, error) in
             UNUserNotificationCenter.current().getNotificationSettings { settings in
                 completion(settings.authorizationStatus)
```

Comment:
Discussed in https://github.com/LoopKit/Loop/issues/1536 but need to make sure this is called on each startup since we are not persisting it anywhere.